### PR TITLE
Patch goreleaser action

### PR DESCRIPTION
**Pull request by AnomalRoil**
RELEASE_NOTES=n/a

Signed-off-by: Yolan Romailler <yolan@romailler.ch>

### CODE REVIEW

File: helpers/changelog/main.go [hunk 1]
```diff
@@ -24,6 +29,11 @@ func main() {
...
 			in = true
 		}
-		fmt.Println(line)
+
+		_, err := fw.WriteString(line + "\n")
+		if err != nil {
+			panic(err)
+		}
 	}
+	fw.Sync()
```
Comment (dominikschulz) [inline helpers/changelog/main.go:38]: I've never used this method before. I don't think it's necessary.
But you can leave it as is if you want.

File: helpers/changelog/main.go [hunk 2]
```diff
@@ -24,6 +29,11 @@ func main() {
 			}
 			in = true
 		}
-		fmt.Println(line)
+
+		_, err := fw.WriteString(line + "\n")
```
Comment (dominikschulz) [inline helpers/changelog/main.go:33]: I'd rather use `fmt.Fprintln(fw, line)`, that's more common.

File: helpers/changelog/main.go [hunk 3]
```diff
@@ -14,6 +13,12 @@ func main() {
 	}
 	defer fh.Close()
 
+	fw, err := os.Create("RELEASE_NOTES")
```
Comment (dominikschulz) [inline helpers/changelog/main.go:16]: Hmm, if someone runs this helper locally in a checkout they might end up committing this along their changes.
But on the other hand I'm happy to argue if someone runs these helpers they should know what they are doing.

**Review by dominikschulz — APPROVED** [review]
LGTM. Some nits, but nothing blocking.

File: helpers/changelog/main.go (hunk 3 above)
Comment (AnomalRoil) [reply helpers/changelog/main.go:17]: Yeah, let's try like this and see if we can figure out another way later.

Comment (dominikschulz) [reply helpers/changelog/main.go:17]: sg

**Review by dominikschulz — APPROVED** [review]
LGTM

**Pull request merged**